modules/appeals_api/spec/requests/appeals_request_spec.rb

```diff
@@ -51,6 +51,63 @@
     end
   end
 
+  describe 'POST /higher_level_reviews' do
+    context 'with an accepted response' do
+      it 'higher level review endpoint returns a successful response' do
+        VCR.use_cassette('decision_review/202_review') do
+          request = {
+            'data' => {
+              'type' => 'HigherLevelReview',
+              'attributes' => {
+                'receiptDate' => '2019-07-10',
+                'informalConference' => true,
+                'sameOffice' => false,
+                'legacyOptInApproved' => true,
+                'benefitType' => 'compensation',
+                'veteran' => {
+                  'fileNumberOrSsn' => '123456789',
```

Comment:
Stylistically, not a fan of `fileNumberOrSsn` - are we allowed to modify this API?

Reply:
We are just passing the data that the decision review API expects. You'd have to discuss that with either @erluti or @edmangimelli